lib/navigator.dart

```diff
@@ -75,4 +80,12 @@ class AppNavigatorImpl extends AppNavigator {
         _routeQuestions,
         arguments: surveyDetail,
       );
+
+  @override
+  void navigateToCompletion(BuildContext context, String? outroMessage) =>
```

Comment:
Would you mind if I fix it in https://github.com/chornerman/Survey/pull/116 to avoid tons of conflict 🙏🏼 I will update the screen name to `SurveyCompletion` as well

Reply:
@markgravity Fixed in https://github.com/chornerman/Survey/pull/116/commits/974bf470fb8f4299dd1d7a0922a940600a09d9c7